src/metabase/query_processor/middleware/metrics.clj

```diff
@@ -114,7 +123,7 @@
                                  new-joins)]
     (lib.util/update-query-stage query-with-joins agg-stage-index add-join-aliases source-field->join-alias)))
 
-(defn splice-compatible-metrics
+(defn- splice-compatible-metrics
```

Comment:
Unrelated, but AFAICT nothing outside this namespace was using it, so I made it private.